client/src/Login.js

```diff
@@ -50,12 +93,37 @@ function Login() {
                 item
                 xs={6}
                 style={{backgroundColor: '#444449', minWidth: '90%', minHeight: '90%', margin: 'auto', alignItems:"center"}}>
-                <p className={classes.mainContentStyle}>Login</p>
+                <p className={classes.mainContentStyle}>{buttonText}</p>
                 <p className={classes.SubHeadings} style={{marginBottom:'50px'}}>Welcome to CloudHaven</p>
                 <Grid container justify="center" alignItems="center">
-                  <TextField color="whitesmoke" className="textfield" label='username' variant="filled"/>
-                  <TextField color="whitesmoke" className="textfield" label='password' variant="filled"/>
-                  <Button variant="contained" color="primary" href="" className={classes.Button}>Login</Button> 
+                  {/* <TextField color="secondary" className="textfield" label='Username' variant="filled"/> */}
+                  <CssTextField
+                    label="Username"
+                    variant="outlined"
+                    id="custom-css-outlined-input"
+                    />
+                  {/* <TextField color="whitesmoke" className="textfield" label='Password' variant="filled"/> */}
```

Comment:
When you click into the password text field the help text overlaps with the username textbox. 
![Overlap](https://user-images.githubusercontent.com/38015797/108887692-6e05a880-75bf-11eb-90eb-2f383752ed57.JPG)